src/routes/github/configuration/github-configuration-post.ts

```diff
@@ -70,17 +70,21 @@ export const GithubConfigurationPost = async (req: Request, res: Response): Prom
 			return;
 		}
 
-		const subscription = await Subscription.install({
+		const subscription: Subscription = await Subscription.install({
 			hashedClientKey: req.body.clientKey,
 			installationId: gitHubInstallationId,
 			host: jiraHost,
 			gitHubAppId
 		});
 
+		//mark is as current date so that subsequent backfill will override it
+		subscription.backfillSince = new Date();
```

Comment:
So for all new github org connection ( subscription ), the backfillSinceDate is mark as current date():

1. Before turn on `sync-main-commit-time-limit`, the `commitSince` will be `undefined`, upon successful backfill, it will override this.
2. After turn on `sync-main-commit-time-limit`, the `commitSince` will be `6 months (or so)`, upon successful backfill, it will override this.
3. On the UI, user will NOT see anything until the backfill is complete and success.

So I think this `subscription.backfillSince = new Date();` is safe. 

The purpose is, it help the code to understand, whether the subscription is a new backfill or existing with undefined backfillSinceDate.